wpeverest/everest-forms
# Tweak - Meta key field readonly

### All Submissions:






### Changes proposed in this Pull Request:

This PR makes the meta key field readonly.

Closes # .

### How to test the changes in this Pull Request:

1. Go to **All Forms** and edit the form.
2. Click on any field and check whether the meta key field is readonly or not.

### Types of changes:




### Other information:




### Changelog entry

> Tweak - Meta key field readonly.
